pyclashbot/emulators/google_play.py

```diff
@@ -17,7 +17,8 @@
 
 
 class GooglePlayEmulatorController(BaseEmulatorController):
-    def __init__(self, render_settings: dict = {}):
+    def __init__(self, render_settings: dict = {}, logger=None):
```

Comment:
What if we made the logger enforced in this signature so we don't have to check for its existence every time?